cmd/namespace-lock.go

```diff
@@ -120,8 +112,28 @@ type nsParam struct {
 
 // nsLock - provides primitives for locking critical namespace regions.
 type nsLock struct {
-	RWLockerSync
-	ref uint
+	DRWMutexLock *dsync.DRWMutex
+	LRWMutexLock *lsync.LRWMutex
```

Comment:
You can update to latest master you won't need this change @wlan0

Reply:
Done